app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -255,18 +288,19 @@ class WeightedQueryStrategy
       def initialize(user: nil, number_of_articles: 50, page: 1, tag: nil, **config)
         @user = user
         @number_of_articles = number_of_articles.to_i
-        @page = page.to_i
+        @page = (page || 1).to_i
```

Comment:
this is to handle the case where an explicit `page: nil` keyword arg is passed to initialize?

Is this the same reasoning as the feeds controller change to set to 1 if not provided (to avoid sending nil to begin with), just being defensive in a second location?